src/data/faq.json

```diff
@@ -101,6 +101,170 @@
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vac_cert_privileges_travel",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "",
+                        "anchor": "vac_cert_how",
+                        "active": true,
+                        "textblock": [
+                            ""
+                        ]
+                    },
+                    {
+                        "title": "Can I show the QR code without an internet connection?",
+                        "anchor": "vac_cert_internet",
+                        "active": true,
+                        "textblock": [
+                            "Yes, indeed. As soon as the QR code is displayed in the Corona-Warn-App, you can show it at any time without being connected to the internet."
+
+                        ]
+                    },
+                    {
+                        "title": "Am I obliged to use the digital certificate of vaccination in the Corona-Warn-App? Will there still be the yellow analog vaccination booklet? Can I continue to use my yellow vaccination booklet? What about people who don't own a smartphone?",
+                        "anchor": "vac_cert_voluntary",
+                        "active": true,
+                        "textblock": [
+                            "The digital proof of vaccination via the Corona-Warn-App is a completely voluntary and complementary offer. If you do not have a digital certificate of vaccination or have lost it, the proof of vaccination via the well-known 'yellow booklet' (yellow vaccination passport) is still possible and valid. You can also prove Corona vaccinations with the printed Corona vaccination certificate for full vaccination."                        ]
+                    },
+                    {
+                        "title": "How is the proof of vaccination verified?",
+                        "anchor": "vac_cert_verification",
+                        "active": true,
+                        "textblock": [
+                            "The vaccination certificate (QR code) is used digitally via the CovPass app or the Corona-Warn-App (CWA) or alternatively as a machine-readable print. The vaccination certificate contains only information about the vaccination status, the name of the vaccinated person, and the date of birth. For service providers who want to check the vaccination status, there will be a verification app to check the vaccination certificate. This allows the vaccination status to be scanned in a similar way to a barcode of a flight or train ticket. Alternatively, proof with the analogue vaccination document is possible."

```

Comment:
"analogue" => "analog" to be consistent with other uses of the word and to use US English.

Reply:
2x